lib/intercom/index.js

```diff
@@ -20,12 +18,10 @@ var is = require('is');
  */
 
 var Intercom = module.exports = integration('Intercom')
-  .assumesPageview()
```

Comment:
We don't trigger an update on load, so without this change a single `page` call is ignored.